WooCommerce/Classes/ViewRelated/Products/ProductLoaderViewController.swift

```diff
@@ -162,23 +176,29 @@ private extension ProductLoaderViewController {
     }
 
     func presentProductDetails(for product: Product, isEditProductsEnabled: Bool) {
-        let viewController: UIViewController
         if product.productType == .simple && isEditProductsEnabled {
-            viewController = ProductFormViewController(product: product, currency: currency)
+            let productForm = ProductFormViewController(product: product, currency: currency) { viewController in
+                viewController.dismiss(animated: true, completion: nil)
+            }
+            productFormViewController = productForm
+
+            productForm.navigationItem.hidesBackButton = true
+            productForm.addCloseNavigationBarButton(target: self, action: #selector(productFormCloseButtonTapped))
+
+            navigationController?.pushViewController(productForm, animated: false)
```

Comment:
Unfortunately pushing the view controller here creates a problem if you try to do a swipe back. I tried to use  
```
navigationController?.setViewControllers([productForm], animated: true)
```
but it seems that in this case, the close button doesn't work anymore 🤔 

<img src="https://user-images.githubusercontent.com/495617/80999950-c9568680-8e45-11ea-8947-c4c29e96586c.png" width=300 />